package.json

```diff
@@ -58,5 +58,6 @@
     "prettier": "^2.8.8",
     "publint": "^0.1.12",
     "shx": "^0.3.4"
-  }
+  },
+  "dependencies": {}
```

Comment:
This looks like a leftover. `npm install`ing Comlink should fix it.

Reply:
I'm actually not depending on it as I kind of need to use the cdn else we would need to package it somehow. Will remove this

Reply:
See my comment elsewhere. We want to depend on Comlink. Exception is the demo.

Reply:
Oups I had not seen your other comment. Will take a complete look tomorrow.

Reply:
Done.